src/main/java/org/spongepowered/api/block/data/Sign.java

```diff
@@ -37,7 +37,7 @@
      *
      * @return The lines of text
      */
-    Message[] getLines();
+    Text[] getLines();
```

Comment:
Since Sign is block data, it is far closer to implementation that the rest of the API. It extends TileEntity, and in consequence, BlockData, meaning that lines get serialized directly to NBT data. This necessitates using the simplest data type possible, meaning Array. Besides, changing this functionality is out of scope of this PR, you should have bothered us on it [here](https://github.com/SpongePowered/SpongeAPI/pull/381/files#diff-6a62facd1257d3f7cb258a2e094b9a7cR40) instead.